src/Element.js

```diff
@@ -136,6 +137,86 @@ export default class Element extends ParentNode {
         });
         return _array;
     }
+
+    /**
+     * Returns the first element that is a descendant of the element on which it is invoked that matches the
+     * specified group of selectors.
+     * @param {String} query CSS query for selecting element
+     * @return {Element} reference to an Element, or null if element was not found
+     */
+    querySelector(query) {
+        let parser = new CssSelectorParser.CssSelectorParser();
+        let rules = parser.parse(query);
+        var result = null;
+
+        /**
+         * Function process all selectors
+         * @param element
+         * @param selectors
+         * @returns {boolean}
+         */
+        var processSelectors = (element, selectors) => {
+            return selectors.every((selector) => {
+                processRule(element, selector.rule);
+            });
+        };
+
+        /**
+         * Function process one rule on element
+         * @param element
+         * @param rule
+         * @returns {boolean}
+         */
+        var processRule = (element, rule) => {
+
+            /**
+             * Function processes one element using current rule
+             * @param element
+             * @returns {boolean|false}
```

Comment:
Yep, and **JSCS** says **return false** is wrong return type. Tried several ways stoped on **return !true** (sic). Forgot to remove **false** here.